src/Module.zig

```diff
@@ -350,7 +350,7 @@ pub const Decl = struct {
     /// Reference to externally owned memory.
     /// In the case of the Decl corresponding to a file, this is
     /// the namespace of the struct, since there is no parent.
-    namespace: *Scope.Namespace,
+    src_namespace: *Scope.Namespace,
```

Comment:
This name was changed because the semantics have changed -- most uses should now use `block.namespace` instead.  I want this to cause compile errors on other PRs if they are using the old semantics.